cryptostock/celery_tasks/broker_notification_tasks.py

```diff
@@ -15,8 +16,10 @@ def notification_success_offer(offer_id):
         f"bought {offer.count} {offer.deal.asset.name} in {offer.timestamp}."
     )
     recipient = [str(offer.deal.broker.owner.email)]
-
-    print(f"{subject}\n{message}\n{recipient}")
+    notification(
+        chat_id=offer.broker.telegram_chat_id,
+        text=f"{subject}\n{message}\nbuyer email: {recipient}",
+    )
```

Comment:
I think signature for `notify` should be producer independent. 
```suggestion
    notification(
        broker=offer.broker,
        text=f"{subject}\n{message}\nbuyer email: {recipient}",
    )
```

Reply:
Also notification is async method.

Reply:
but for this function I need exactly id